### ISSUE TITLE
compillation errors with catkin_make

### ISSUE THREAD

**Issue by MarcelSoler** [SteveMacenski/spatio_temporal_voxel_layer#99]
On ubuntu16.06 kinetic with cakin_make:

```
CMake Error at spatio_temporal_voxel_layer/CMakeLists.txt:41 (find_package):
  By not providing "FindOpenVDB.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "OpenVDB", but
  CMake did not find one.

  Could not find a package configuration file provided by "OpenVDB" with any
  of the following names:

    OpenVDBConfig.cmake
    openvdb-config.cmake

  Add the installation prefix of "OpenVDB" to CMAKE_PREFIX_PATH or set
  "OpenVDB_DIR" to a directory containing one of the above files.  If
  "OpenVDB" provides a separate development package or SDK, be sure it has
  been installed.
```
And with catkin build it works.
Thank you,

**Comment by SteveMacenski** [comment]
Did you rosdep install your dependencies like in the instructions? 
This compiles in CI and our machines, please make sure you have all the dependencies required through rosdep

**Comment by SteveMacenski** [comment]
Here's the rosdep install instructions. I should frankly remove any other install instructions, I find that people installing them independently has issues. https://github.com/SteveMacenski/spatio_temporal_voxel_layer#install-from-source

**Comment by MarcelSoler** [comment]
I have all dependencies installed with rosdep using` catkin build` compiles. But if i try to use `catkin_make` it doesn't find OpenVDB.

**Comment by nickvaras** [comment]
Just to chip in, I saw a similar thing when I tried to put stvl as a third party subfolder within a larger workspace. In the end I had to give it its own workspace and, after making, sourcing stvl separately. Not sure if related, but maybe.

**Comment by SteveMacenski** [comment]
Can you try changing the line 

`set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/") `

to 

`set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/cmake/")`

to see if that resolves @nickvaras @MarcelSoler

**Comment by MarcelSoler** [comment]
Solved! I just commit it in pr #98
